src/harness/reference_models/pre_iap_filtering/testdata/fss_0.json

```diff
@@ -0,0 +1,31 @@
+{
+  "record": {
+  "id": "incumbent/ibfs/0",
+  "type": "FSS",
+  "deploymentParam": 
+  [
+    {
+      "installationParam": {
+        "latitude": 40.5029,
+        "longitude":  -101.47015,
+        "height": 7.6,
+        "elevation": 100.0,
+        "heightType": "AGL",
+        "antennaGain": 16,
+        "antennaBeamwidth": 30,
+        "antennaAzimuth": 270,
+        "antennaDowntilt": -90,
+        "antennaElevationAngle":39
```

Comment:
I thought this was fixed before, so the issues have come back again somewehat:
 - elevation: this is the terrain height - UNUSED, and should not be in the json
- antennaElevationAngle: the vertical point direction is handled by  antennaDowntilt, as I believe was decided previously.

Reply:
changed